bundles/org.eclipse.text.quicksearch/src/org/eclipse/text/quicksearch/internal/ui/QuickSearchDialog.java

```diff
@@ -261,10 +262,12 @@ public void update(ViewerCell cell) {
 
 	private Image getBlankImage() {
 		if (blankImage==null) {
-			blankImage = new Image(Display.getDefault(), 1, 1);
-//			GC gc = new GC(blankImage);
-//			gc.fillRectangle(0, 0, 16, 16);
-//			gc.dispose();
+			final ImageGcDrawer imageGcDrawer = (gc, width, height) -> {
+//				GC gc = new GC(blankImage);
+//				gc.fillRectangle(0, 0, 16, 16);
+//				gc.dispose();
+			};
+			blankImage = new Image(Display.getDefault(), imageGcDrawer, 1, 1);
```

Comment:
Remove the commented code, there is no value in it. The drawer is effectively a `NOOP`, rename accordingly:

```suggestion
			final ImageGcDrawer noOp = (gc, width, height) -> {};
			blankImage = new Image(Display.getDefault(), noOp, 1, 1);
```

Reply:
Done in b2a7f24c75c7d530d84be3ed2bc58a8ea1fcb383